templates/Web/Projects/VueDefault/README_postaction.md

```diff
@@ -0,0 +1,56 @@
+﻿## Getting Started
+
+In the root directory of the project...
+
+1. Install node modules `yarn install` or `npm install`.
+2. Start development server `yarn start` or `npm start`.
+
+## Next Steps
+//{[{
```

Comment:
Are these brackets supposed to be here

Reply:
Good question, that's an insertion macro essentially states everything between the //{[{ and //}]} is going to be inserted within the context of whats above the //{[{ and below //}]} so in this case it would insert 

```
### Adding a New Page

 1. Create a file in `/src/views` with your Vue Template.
2. Add a route for your page to `/src/router/index.js`.
3. Add a button to the navigation bar in `/src/components/NavBar.vue`.
```

Between
```
 ## Getting Started

 In the root directory of the project...

 1. Install node modules `yarn install` or `npm install`.
2. Start development server `yarn start` or `npm start`.

 ## Next Steps
```

and

```
### Deployment

 The generated templates can be deployed to Azure App Service using the following steps:

 1. In the root directory of the project `yarn build` or `npm build` to create a build folder.
2. Move the build folder inside the server folder.
3. Deploy the server folder to Azure App Service using the [Azure App Service Extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice).
4. If a database is used, add the environment variables defined in .env to your Application Settings.
5. Consider adding authentication and securing back-end API's by following [Azure App Service Security](https://docs.microsoft.com/en-us/azure/app-service/overview-security).
   Full documentation for deployment to Azure App Service can be found here: [Deployment Docs](https://github.com/Microsoft/WebTemplateStudio/blob/dev/docs/deployment.md).
.
.
.
```

Reply:
I see, that's pretty cool. I assume `//^^` is a similar macro?

Could you refer me to where the insertion happens?

Reply:
Yup exactly //^^ means insert closer to bottom macro //}]} by default its closer to the {[{